Docs/Architecture/CommandBasedPresentationNotes.md

```diff
@@ -27,25 +28,37 @@ A recording should be posted once the presentation is completed.
 
 ### How to make Commands
 
-- The two main types of Commands we use are `RunCommand`s and `InstantCommand`s
-  - `RunCommand`s run an action over and over until they are interrupted
-  - `InstantCommand`s run once and then immediately stop
+- The two main types of Commands we use are `run`s and `runOnce`s
+  - `run` runs an action over and over until it is interrupted
+  - `runOnce` runs once and then immediately stops
```

Comment:
Are we standardizing this? Maybe make a note of phasing out (for example) `RunCommand` in favor of `Commands.run()` syntax (since it's cleaner imo).

Reply:
whoops branched from the wrong branch, this is supposed to just be in the other pr